pkgs/top-level/python-packages.nix

```diff
@@ -14066,11 +14066,11 @@ in modules // {
   };
 
   mock = buildPythonPackage (rec {
-    name = "mock-1.3.0";
+    name = "mock-2.0.0";
```

Comment:
OK, build finished, and I restarted a few times to make sure I got a stable point. I now get these errors:

https://gist.github.com/teh/d1d4fe3bcae794581abea0d6adbaf329

But bizarrely when I pick out individual ones they build fine. Do you have any way of approaching this? I could of course add `mock-2_0_0` but at some point we probably have to make the jump, no?